pkg/vpa/vpa_test.go

```diff
@@ -332,3 +339,70 @@ func Test_ReconcileNamespaceRemoveLabel(t *testing.T) {
 	assert.Equal(t, 0, len(vpaList.Items))
 	assert.EqualValues(t, vpaList, &v1beta2.VerticalPodAutoscalerList{})
 }
+
+func Test_ReconcileNamespace_ExcludeDeploymentAnnotation(t *testing.T) {
```

Comment:
I love the test harness that y'all set up btw. It makes it _very_ easy to add tests and ensure functionality!